utils/license_utils.jsx

```diff
@@ -13,6 +13,17 @@ export function isLicenseExpiring(license) {
     return timeDiff <= LICENSE_EXPIRY_NOTIFICATION;
 }
 
+export function isLicenseExpiringIn(license, days) {
```

Comment:
See above, but may be better to just determine `daysToLicenseExpire`.

Reply:
Gooooood call. Made for a cleaner implementation too.